spinalcordtoolbox/image.py

```diff
@@ -1603,3 +1603,33 @@ def _align_dict(dictionary, use_tabs=True, delimiter=""):
             padding = " " * (len_max - len(k))
         out.append(f"{k}{padding}{delimiter}{v}")
     return '\n'.join(out)
+
+
+def apply_mask_if_soft(fname_im: str, fname_mask: str):
```

Comment:
Would it be better to put this in `register.py` rather than `image.py`? This would further emphasize that this is a hacky workaround specific to registration, rather than an image processing function (e.g. this shouldn't be used in `sct_image`).

Reply:
Yes I agree that the workaround is better defined locally in `register.py`.